cmd/create/cluster/cmd.go

```diff
@@ -225,12 +225,13 @@ func init() {
 		"Simulate creating the cluster.",
 	)
 
-	flags.StringArrayVar(
+	flags.StringSliceVar(
 		&args.subnetIDs,
 		"subnet-ids",
 		nil,
 		"The Subnet IDs to use when installing the cluster. "+
 			"SubnetIDs should come in pairs; two per availability zone, one private and one public. "+
+			"Subnets are comma separated, for example: --subnet-ids='subnet-1','subnet-2'."+
```

Comment:
Remove the single quotes, they should not be necessary once you fix the `Default` used for pre-populating the `GetMultipleOptions` element: `--subnet-ids=subnet-1,subnet-2`:
```
$ rosa create cluster -i --dry-run --subnet-ids=subnet-050b5a070768d2737,subnet-074572b18f5698f1f
I: Interactive mode enabled.
Any optional fields can be left empty and a default will be selected.
? Cluster name: foo
? Multiple availability zones (optional): No
? AWS region: us-east-1
? OpenShift version (optional): 
? Subnet IDs:  [Use arrows to move, space to select, <right> to all, <left> to none, type to filter, ? for more help]
> [x]  subnet-050b5a070768d2737 (us-east-1f)
  [x]  subnet-074572b18f5698f1f (us-east-1a)
  [ ]  subnet-09a60d10e113314e5 (us-east-1a)
  [ ]  subnet-011158850f4d15a53 (us-east-1a)
  [ ]  subnet-0ba3f0ead1f486510 (us-east-1a)
  [ ]  subnet-0c031ef9162b35f04 (us-east-1a)
  [ ]  subnet-0c7b531f9375931b1 (us-east-1f)
```